src/App.jsx

```diff
@@ -1,3 +1,42 @@
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Home from './views/Home/Home';
+import Auth from './components/Auth/Auth.jsx'
+import Header from './components/Header/Header';
+import { UserProvider } from './context/UserContext/UserContext';
+import ConfirmEmail from './components/Auth/ConfirmEmail';
+import Profile from './views/Profile/Profile';
+import CreateProfile from './views/Profile/CreateProfile';
+import PrivateRoute from './components/PrivateRoute/PrivateRoute';
+
 export default function App() {
-  return <h1>Hello World</h1>;
+  return (
+    <UserProvider>
```

Comment:
Nice job using your provider to hold the user's info. I really like your css color scheme and layout. Code files are clean and readable